lib/features/reps_core/reps_core.views_default.inc

```diff
@@ -11,252 +11,6 @@
 function reps_core_views_default_views() {
   $export = array();
 
-  $view = new view();
-  $view->name = 'reps_bean_blocks';
```

Comment:
Why deleting this view if we apply a query alter to it ?

Reply:
no the name is now reps_bean_blocks_global